operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/ReconciliationDispatcher.java

```diff
@@ -60,7 +59,7 @@ public ReconciliationDispatcher(Controller<R> controller) {
     log.debug("Handling dispatch for resource {}", getName(resource));
 
     final var markedForDeletion = resource.isMarkedForDeletion();
-    if (markedForDeletion && shouldNotDispatchToDelete(resource)) {
+    if (markedForDeletion && shouldNotDispatchToCleanup(resource)) {
       log.debug(
           "Skipping delete of resource {} because finalizer(s) {} don't allow processing yet",
```

Comment:
another naming discrepancy delete/cleanup

Reply:
plus another log message in handleCleanup out of the scope review

Reply:
fixed.